Unable to install with npm strict engine

**package**: v1.7.2
**npm**: v6.14.17
**node**: 16.15.0

```
npm install @strapi/plugin-seo --save

npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for @strapi/plugin-seo@1.7.2: wanted: {"node":">=12.x. <=16.x.x","npm":">=6.0.0"} (current: {"node":"16.15.0","npm":"6.14.17"})
npm ERR! notsup Not compatible with your version of node/npm: @strapi/plugin-seo@1.7.2
npm ERR! notsup Not compatible with your version of node/npm: @strapi/plugin-seo@1.7.2
npm ERR! notsup Required: {"node":">=12.x. <=16.x.x","npm":">=6.0.0"}
npm ERR! notsup Actual:   {"npm":"6.14.17","node":"16.15.0"}
```

`.npmrc` configuration
```
engine-strict=true
```

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
     }
   ],
   "engines": {
-    "node": ">=12.x. <=16.x.x",
+    "node": ">=12.x.x <=16.x.x",
     "npm": ">=6.0.0"
   },
   "license": "MIT"

